src/main/java/org/embulk/gradle/runset/InstallEmbulkRunSet.java

```diff
@@ -234,11 +248,47 @@ private static Map<Object, Object> castMap(final Map map) {
         return (Map<Object, Object>) map;
     }
 
+    @Override
+    @TaskAction
+    protected void copy() {
+        if (this.embulkSystemPropertiesSource != null) {
+            try (final OutputStream out = Files.newOutputStream(this.embulkSystemPropertiesSource)) {
+                this.embulkSystemProperties.store(
+                        out, "Generated by the \"org.embulk.embulk-runset\" Gradle plugin.");
+            } catch (final IOException ex) {
+                throw new UncheckedIOException(ex);
+            }
+        }
+        super.copy();
+    }
+
+    private synchronized void createPropertiesSourceAndSetToCopy() {
+        if (this.embulkSystemPropertiesSource == null) {
+            final File temporaryDir = this.getTemporaryDir();
+            final Path path;
+            try {
+                path = Files.createTempFile(temporaryDir.toPath(), "embulk.", ".properties");
+            } catch (final IOException ex) {
+                throw new UncheckedIOException(ex);
+            }
+
+            this.from(path.toFile(), copySpec -> {
+                copySpec.rename(path.getFileName().toString(), "embulk.properties");
```

Comment:
Does this part work if a temporary directory and `$HOME` use the different file partitions?
(i.e. `/home` and `/var/tmp` use the different partitions)

man from `rename(2)`

```
       EXDEV  oldpath and newpath are not  on  the  same  mounted  filesystem.
              (Linux  permits  a  filesystem to be mounted at multiple points,
              but rename() does not work across different mount  points,  even
              if the same filesystem is mounted on both.)
```

Reply:
This `rename` does NOT mean the 
`rename` system call. It's just Gradle's implementation in `Copy`.

Reply:
Thanks!